paparazzi/src/main/java/app/cash/paparazzi/internal/resources/ResourceFolderRepository.kt

```diff
@@ -40,7 +40,7 @@ internal class ResourceFolderRepository(
    * Common prefix of paths of all file resources.  Used to compose resource paths returned by
    * the [BasicFileResourceItem.getSource] method.
    */
-  private val resourcePathPrefix: String = "${resourceDir.path}/"
+  private val resourcePathPrefix: String = "${resourceDir.path}${File.separator}"
```

Comment:
this, was causing a `FileNotFoundException` on Windows. Needed to respect the correct file separator on the platform

Reply:
Actually, Android Studio does fine with "/": https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:android/src/com/android/tools/idea/res/ResourceFolderRepository.java;l=275?q=ResourceFolderRepository

The ResourceRepository mechanism is separate from the ASM compat stuff they load.  Let's investigate where that change is being made.

Reply:
Ya I think the issue stems from our internal resource code we pulled from Android Studio code. I can dig into why folder structure changes for windows

Reply:
So, once I set up Parallels on my Mac, I was able to debug this and it is now fixed! We don't need this change after all!